resources/ts/Components/FormElement.tsx

```diff
@@ -65,9 +65,7 @@ const InputStyle = {
 };
 
 type Props = {
-  label?: string;
-  clickableLabel?: string;
-  clickableLabelLink?: string;
+  label?: string | ReactElement;
```

Comment:
Having two totally different types is mostly a code smell. Maybe this combined field is no longer a label but the component itself?

Reply:
I think `ReactElement` even contains the string type. But would have to check.
For a label I can see the use case where you most of the time want a simple string that will be injected into a label element.
But sometimes we need more control over the element injected into the label. e.g. setting html content with custom link components as it is used here.

Reply:
I looked it up

`ReactElement` doesn't contain string
`ReactNode` does (and all other options react could render)

So hande's type seems the most restrictive option